src/baldr/directededge.cc

```diff
@@ -586,6 +596,8 @@ json::MapPtr DirectedEdge::json() const {
       {"destination_only", static_cast<bool>(dest_only_)},
       {"tunnel", static_cast<bool>(tunnel_)},
       {"bridge", static_cast<bool>(bridge_)},
+      {"is_hov", static_cast<bool>(is_hov_only())},
+      {"hov_type", to_string(static_cast<HOVEdgeType>(hov_type_))},
```

Comment:
nit: non hov edges return the string "null" which is kind of odd for json. instead of doing it inline you can put an if below that says `if(is_hov_only) map.emplace("hov_type", to_string(static_cast<HOVEdgeType>(hov_type_)));` so that it only shows up when its not "null"